tsfile/src/test/java/org/apache/iotdb/tsfile/read/filter/IExpressionOptimizerTest.java

```diff
@@ -67,7 +66,7 @@ public void testTimeOnly() {
           new GlobalTimeExpression(TimeFilter.gt(200L)));
 
     } catch (QueryFilterOptimizationException e) {
-      e.printStackTrace();
+      Assert.fail();
```

Comment:
`TsFileSequenceReaderTest.java` test bug doesn't occur in here. This is the change request from @jixuan1989 in [#389](https://github.com/apache/incubator-iotdb/pull/389). He think I should change `e.printStackTrace()` to `logger.error()`, but I think `Assert.fail()` is better because using log should create a new log object. BTW, [#389](https://github.com/apache/incubator-iotdb/pull/389) is closed by me accidental.

Reply:
I agree. More importantly, logs may be ignored, but fails will not and we should not ignore such situations.